.gitignore

```diff
@@ -1,11 +1,17 @@
-*.iml
+
 *.log
 *.swp
 .classpath
 .gradle/
-.idea/
 .project
 .settings/
 /bin
 bin/
-build/
+build
```

Comment:
Yes. This will ignore all files / directories / sub directories named "build" as opposed to just directories / sub directories named "build". Personally I haven't seen code / files added with the name "build". They are mostly auto-generated by build tools. If you recommend reverting this, i'll do it.